lib/dogma/rules/final_newline.ex

```diff
@@ -8,7 +8,9 @@ defmodule Dogma.Rules.FinalNewline do
   alias Dogma.Script
   alias Dogma.Error
 
-  def test(script) do
+  def test(script), do: test(script, [])
+
+  def test(script, _config = []) do
```

Comment:
We could get rid of `test/1` and always have rules called with two functions, then it's in one place.

There's a bunch of little refactorings I want to do, but I've just started a new job, so I don't have the time at the moment I'm afraid!